app/scripts/components/exploration/components/datasets/data-layer-card.tsx

```diff
@@ -317,3 +322,41 @@ export default function DataLayerCard(props: CardProps) {
     </>
   );
 }
+
+export function DataLayerCardWithSync(props: {
```

Comment:
Since this is a wrapper specific to the exploration page, should we move it there and keep the data-layer-card component clean?

Reply:
I was thinking this can be used outside of E&A page, ex. water insight instance can use this component directly. 
Based on the convo we had today, I do think users (like dev users of veda-ui) can use the hooks (that basically this component is using: https://github.com/NASA-IMPACT/veda-ui/pull/1801/files/f6087fdbafdb62e326b10c1c44f4a609fbae1c99#diff-9ac16b2c33a88f83221cfa1b63bccc5eaa8ff468a605c0fe07d6f62f255c06a7R334)  and compose their own UIs too.  But I do think there is a value of offering this level of components to dev users, like offering already-composed components that use the hooks. I will talk more about this with water insight team today.